build.gradle

```diff
@@ -23,7 +23,6 @@ allprojects {
         jcenter()
         maven { url 'https://oss.sonatype.org/content/groups/public' }
         maven { url 'https://jitpack.io' }
-        maven { url 'https://staging.dev.medicmobile.org/_couch/maven-repo' }
```

Comment:
This repo link doesn't seem to be working any longer as they don't provide a `.pom` for the dependency. Given its a pretty stagnant dependency I think using a local jar is fine.

Reply:
I think such a trick would work:
```
maven {
            url 'https://staging.dev.medicmobile.org/_couch/maven-repo'
            metadataSources { artifact() }
        }
```
and you won't need anything more.

Reply:
I was looking around for something like this as it really surprised me that there wasn't a way to get it working with just a JAR. Will give this a go tomorrow!

Reply:
Great if we can track the Medic repo but not super important I don’t think.

Reply:
Yes I was right. Just added a commit on my own, so that you don't need to play with it just take a look.